### PR TITLE
Display statistic map even if excursion set is empty

### LINKED ISSUE #9: Cannot read property 'substr' of undefined
I'm getting an error with those two NIDM-Results 1.3.0 examples:
http://neurovault.org/collections/1404/fsl_ds005_sub-01.nidm
http://neurovault.org/collections/1404/fsl_ds005_group.nidm

Uncaught TypeError: Cannot read property 'substr' of undefined
papaya.volume.Volume.readURLs  @ papaya.js:399
papaya.viewer.Viewer.loadOverlay @ papaya.js:648
(anonymous function) @ fsl_ds005_sub-01.nidm:205

### PR DESCRIPTION
**Pull request by cmaumet**
This PR fixes #9 by enabling display of a statistic map in the absence of suprathreshold voxels.

Tested on my machine the following command:

```
nidmviewer PATH_TO_EXAMPLE/fsl_ds005_group/nidm.ttl
```

returns:
image

### CODE REVIEW

**Comment by cmaumet** [comment]
(Note as FSL does not currently return the list of peaks and clusters for voxel-wise thresholds, the list of peaks can be empty even if the excursion set is not).

File: nidmviewer/convert.py [hunk 1]
```diff
@@ -24,7 +25,10 @@ def parse_coordinates(coordinates):
     count=0
     coordinate_df = pandas.DataFrame(columns=["x","y","z"])
     for coordinate in coordinates:
-        coordinate_df.loc[count] = [x.strip() for x in coordinate.strip("]").strip("[").split(",")]
+        if not isinstance(coordinate, float64) or not isnan(coordinate):
+            coordinate_df.loc[count] = [x.strip() for x in coordinate.strip("]").strip("[").split(",")]
+        else:
```
Comment (vsoch) [inline nidmviewer/convert.py:30]: This will likely bug out - if we have a general float we will get an error when we assume string and try to strip it.

**Comment by chrisgorgo** [comment]
Since we are moving to displaying excursion sets, wouldn't a better approach be just to display a warning saying that "there are no suprathreshold voxels" and not display any map? It would be more consistent.

**Comment by vsoch** [comment]
What exactly is an excursion set? If it is what I think it is, +1

**Comment by chrisgorgo** [comment]
Excursion set is a fancy name for a thresholded map (as in a nifti file with non-significant voxels set to to zero).

**Comment by cmaumet** [comment]
@chrisfilo: as FSL does not report peaks and clusters for voxel-wise threshold, you might have no peaks and clusters even if the excursion set in non-empty.

**Comment by chrisgorgo** [comment]
That sounds suboptimal and limits the usefulness of NIDM-Results for people doing peak based metaanalysis. 

In the context of the current bug: Is there a way in NIDM-Results to reliably (across FSL, SPM, AFNI) check if there are any suprathreshold voxels without having to read any of the nifri files?

**Comment by cmaumet** [comment]
> That sounds suboptimal and limits the usefulness of NIDM-Results for people doing peak based metaanalysis.

Yes, I agree. We are working with FSL to make this happen. But this is not a limitation of NIDM-Results as any study using a voxel-wise threshold within FSL will have no listing of peaks and clusters.

> In the context of the current bug: Is there a way in NIDM-Results to reliably (across FSL, SPM, AFNI) 
> check if there are any suprathreshold voxels without having to read any of the nifri files?

I don't think there is a way. Shall we continue with the current approach and leave the update to display "no significant clusters" when appropriate for when the issue with FSL is sorted?

File: nidmviewer/convert.py (hunk 1 above)
Comment (cmaumet) [inline nidmviewer/convert.py:30]: Is your suggestion to replace the if condition by

```
if isinstance(coordinate, str):
```

?

**Comment by chrisgorgo** [comment]
Saying "no significant clusters" when the excursion set is not empty is very confusing. I would prefer to avoid this.

@vsoch Can the viewer check if the excursion file includes any nonzero non-nan voxels?

**Comment by cmaumet** [comment]
@chrisfilo: Yes! That's why if we can't know for sure I think we should just not display any message.

**Comment by chrisgorgo** [comment]
Not displaying any message when the table and the excursion set are both empty is also confusing. It looks like the viewer is broken.

**Pull request merged**